### PR TITLE
you can now cast rituals on 100 threat dynamic

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

wizards on 100 threat dynamic can now cast summon guns, magic and events

## Why It's Good For The Game

the round is already chaos and is gonna end in 10 minutes, and the wizard events once in a year would be pretty fun i think

## Changelog
:cl:
add: you can now cast rituals on dynamic if the threat level is 100
/:cl:

### CODE REVIEW

File: code/modules/antagonists/wizard/equipment/spellbook.dm [hunk 1]
```diff
@@ -480,6 +480,9 @@
 	category = "Defensive"
 	cost = 1
 
+/// How much threat we need to let these rituals happen on dynamic
+#define MINIMUM_THREAT_FOR_RITUALS 100
```
Comment (Mothblocks) [inline code/modules/antagonists/wizard/equipment/spellbook.dm:484]: undef at the end

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by nianjiilical** [comment]
make sure theres a notification since most wizards probably arent going to check the rituals page every round on the tiny chance theyre castable

**Comment by necromanceranne** [comment]
Cowabunga it is.

**Comment by oklews** [comment]
This is where we add a button to make the threat level 100 right?

**Comment by RaveRadbury** [comment]
> This is where we add a button to make the threat level 100 right?

**Open** wizard spell to make threat 100

**Review by Mothblocks — APPROVED** [review]

**Review by Timberpoes — APPROVED** [review]

**Pull request merged**